soc/arm/ti_simplelink/cc13x2_cc26x2/power.c

```diff
@@ -23,16 +23,28 @@
 #define LOG_LEVEL CONFIG_SOC_LOG_LEVEL
 LOG_MODULE_REGISTER(soc);
 
+const PowerCC26X2_Config PowerCC26X2_config = {
+#if defined(CONFIG_IEEE802154_CC13XX_CC26XX)
+/* This will need to include BLE eventually */
+	.policyInitFxn      = NULL,
+	.policyFxn          = &PowerCC26XX_doWFI,
+/* PowerCC26XX_standbyPolicy() is not yet available in hal/ti */
```

Comment:
IIRC the .policyFxn is executed without a NULL check somewhere in the RF driver. I wonder if it would be possible to make a "no op" replacement call and, if so, whether that would effectively allow Zephyr to maintain power management control.

Reply:
Would you know where the call site is? I searched through source/ti/drivers/rf/RFCC26X2_multiMode.c in the HAL and did not find direct calls to the policyFxn. It's odd that the RF driver would call the power management policy function.

Reply:
Ah - you're right. I'll mark correct it to NULL. It was the calibrate function that gets executed without a NULL check.